### ISSUE TITLE
Multiple inlines for same model

### ISSUE THREAD

**Issue by jwindhager** [escaped/django-inline-actions#17]
When using multiple inlines for the same model (e.g. with different querysets), django-inline-actions fails in handling the inline actions, because the inline instances are selected based on the model class only, and the action handler function is not defined in all inline instances.

Function `_handle_action(self, request, object_id=None)` in inline_action/admin.py
```Python
for inline in self.get_inline_instances(request):
    if inline.model != model:
        continue
    model_admin = inline
obj = model_admin.get_queryset(request).get(pk=object_pk)
```

A quick fix would be to add an (optional) admin attribute that serves as a unique key, and to include that key value as a hidden field in the form (if defined). Not sure if there is a more elegant solution.

**Comment by escaped** [comment]
Good catch. I guess I have to include the name of the corresponding admin in [action data](https://github.com/escaped/django-inline-actions/blob/589dcafe681293fcecf5eb12db71f4351cb02fee/inline_actions/admin.py#L111). Hopefully I find some time during this week to provide a fix :)

**Comment by escaped** [comment]
Just released version 2.2.0, which includes this feature :)

**Comment by jwindhager** [comment]
Awesome, thank you!
